engine/apps/api/serializers/alert_receive_channel.py

```diff
@@ -562,7 +561,7 @@ def _handle_messaging_backend_updates(self, data, ret):
             for field in TEMPLATE_NAMES_ONLY_WITH_NOTIFICATION_CHANNEL:
                 field_name = f"{backend_id.lower()}_{field}_template"
                 value = data.get(field_name)
-                validator = jinja_template_env.from_string if field != IMAGE_URL else URLValidator()
+                validator = jinja_template_env.from_string
```

Comment:
Do we intentionally want to allow non-URL templates for image URL?

Reply:
It's a template for Image url, not url itself, template will not be valid url anyway.